### ISSUE TITLE
Achievements

### ISSUE THREAD

**Issue by Leirof** [Creative-Olympics/CO2-Datapacks#96]
# TL;DR
In order to push the players to explore all the minigames and enhance the replayability, we imagined an economy system allowing to get some cosmetics. As we have a policy of not reward the performances of the player to avoid cheaters, the way of earning money will pas through personal achievements.

# Description

## Unique achievements
- [ ] Play once each minigame (*N minigames)
- [ ] Play once every minigame
- [ ] Play 10 times each minigame (*N minigames)
- [ ] Play 50 times each minigame (*N minigames)
- [ ] Play 500 times each minigame (*N minigames)
- [ ] Play a total of 100 minigames, all minigames combined
- [ ] Play a total of 1h
- [ ] Play a total of 3h
- [ ] Play a total of 10h

## Hourly achievements

- [ ] Play 3 different minigames in one hour
- [ ] (Prophunt) Stun 3 times a hunter
- [ ] (Prophunt) Switch prop to escape a hunter (switch at less than 5 blocks and survive 10 secondes)
- [ ] (Prophunt) Kill a prop using the dash ability
- [ ] (Dropper) Reach the stage 5
- [ ] (Wallstreet) Make more than 2k benefits (end a game with 7k+)
- [ ] (Jump) Reach the step 10
- [ ] (Jump) Beat your previous high score
- [ ] (Magma Run) Survive more than 30 seconds
- [ ] (One in the chamber) Perform a 3 kill streak

## Daily achievements

- [ ] Win a game (*N minigames)
- [ ] (Prophunt) Kill a prop using the enrage ability
- [ ] (Dropper) Perform a thimble (dé à coudre)
- [ ] (Jump) Finish the jump
- [ ] (Wallstreet) Make a 100+% profit transaction
